save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/SarifParsingException.kt

```diff
@@ -0,0 +1,6 @@
+package org.cqfn.save.core.utils
+
+/**
+ * An [Exception] that can be thrown in case of failed to parse sarif
+ */
+class SarifParsingException(message: String) : Exception(message)
```

Comment:
Please also add `cause` as a parameter (`Exception` has a constructor that accepts cause). We may want to use it in the future for better reporting, so it'll be useful to have it in there already

Reply:
Added